tests/jquery.flot.navigate.Test.js

```diff
@@ -31,18 +31,38 @@ describe("flot navigate plugin", function () {
         expect(typeof plot.zoomOut).toBe('function');
         expect(typeof plot.pan).toBe('function');
         expect(typeof plot.smartPan).toBe('function');
-
-        plot.shutdown();
     });
 
     it('shuts down the plots after use', function() {
         plot = $.plot(placeholder, [
-            []
+            [
+                [0, 0],
+                [10, 10]
+            ]
+        ], options);
+        var canvasElement = placeholder[0].childNodes[2];
```

Comment:
Rename this var eventPlaceholder and instead of taking the second child use the getElementsByClassName() function to find that element. In the future it's more likely to have the order changed rather than class name.